build/docker/intel-deviceplugin-operator.Dockerfile

```diff
@@ -25,7 +25,7 @@ ARG CMD=operator
 ## (see build-image.sh).
 ## 2) and the default FINAL_BASE is primarily used to build Redhat Certified Openshift Operator container images that must be UBI based.
 ## The RedHat build tool does not allow additional image build parameters.
-ARG FINAL_BASE=registry.access.redhat.com/ubi8-micro
+ARG FINAL_BASE=registry.access.redhat.com/ubi8-micro:8.6
```

Comment:
@chaitanya1731 is this selection OK for you?

Reply:
Looks good to me.